CaImAn/caiman_run.py

```diff
@@ -36,6 +36,9 @@
 logging.basicConfig(level=logging.DEBUG)
 freeze_support()
 
+kwargs = {}
+params_doric = {}
+
```

Comment:
I added that because of VScode so kwargs and params_doric are recognized as variable